### PR TITLE
Adds headers for SDK versioning, usage, etc.

### LINKED ISSUE #18: The SDK should supply the user-agent header

### LINKED ISSUE #19: The SDK should supply the legacy headers

### PR DESCRIPTION
**Pull request by iredelmeier**
This includes both the user-agent header and the legacy headers.
Note that it does not change the ClientInfo object or its usage.

Closes #18 and closes #19

### CODE REVIEW

**Comment by iredelmeier** [comment]
The headers are filled in as per discussion with @oderby and @toolateforteddy.

TBD: should the user-agent header include the client ID?

File: flux-sdk-common/spec/unit/utils/request-spec.js [hunk 1]
```diff
@@ -9,8 +9,9 @@ import * as base64 from '../../../src/ports/base64';
 import * as url from '../../../src/ports/url';
 import * as querystring from '../../../src/ports/querystring';
 import credentialsFactory from '../../factories/credentials-factory';
+import { version } from '../../../package.json';
 
-describe('utils.request', function() {
+fdescribe('utils.request', function() {
```
Comment (oderby) [inline flux-sdk-common/spec/unit/utils/request-spec.js]: fdescribe?

**Comment by oderby** [comment]
Have you tested this against prod? I'm uncertain as to the behavior if Flux-Plugin-Host-Version header is missing - I assume it happily accepts it, but we have had enough issues here that I don't actually know if that's true, or if there's some logic which rejects requests which specify Flux-Plugin-Host but not Flux-Plugin-Host-Version. Worth checking that an app using this updated SDK works or not when reading or writing a value to a data key.

**Comment by iredelmeier** [comment]
Yes, this works on prod.

File: flux-sdk-common/spec/unit/utils/request-spec.js [hunk 2]
```diff
@@ -102,7 +103,12 @@ describe('utils.request', function() {
         expect(url.joinUrl).toHaveBeenCalledWith('https://flux.io/', 'SOME_PATH', '');
         expect(fetchPort.fetch).toHaveBeenCalledWith('JOINED_URL', {
           credentials: 'include',
-          headers: { a: 'b' },
+          headers: {
+            a: 'b',
+            'User-Agent': `JsSDK/${version}`,
+            'Flux-Plugin-Version': version,
```
Comment (daishiflux) [inline flux-sdk-common/spec/unit/utils/request-spec.js]: Please don't set this at all.

File: flux-sdk-common/spec/unit/utils/request-spec.js [hunk 3]
```diff
@@ -102,7 +103,12 @@ describe('utils.request', function() {
...
+            'Flux-Plugin-Platform': 'JsSDK',
```
Comment (daishiflux) [inline flux-sdk-common/spec/unit/utils/request-spec.js]: Please set this to the form: 'browser/jssdk/<SDK_VERSION>'.

**Pull request merged**